docker-compose-rule-core/src/main/java/com/palantir/docker/compose/execution/AggressiveShutdownStrategy.java

```diff
@@ -25,42 +22,14 @@
     private static final Logger log = LoggerFactory.getLogger(AggressiveShutdownStrategy.class);
 
     @Override
-    public void shutdown(DockerCompose dockerCompose, Docker docker) throws IOException, InterruptedException {
-        List<ContainerName> runningContainers = dockerCompose.ps();
-
-        log.info("Shutting down {}", runningContainers.stream().map(ContainerName::semanticName).collect(toList()));
-        if (removeContainersCatchingErrors(docker, runningContainers)) {
```

Comment:
This seems like a lot of effort to just do `docker.kill()` - I wonder if there was a good reason behind it?

Reply:
I think the intention was to kill each container individually to do a best-effort shutdown. But given that docker-compose-rule is used almost exclusively for CI builds, this seems unnecessary.

Reply:
Also, I think `docker-compose kill` literally does this.

Reply:
I think you're right